example/gluon/snail/omniglot_dataset.py

```diff
@@ -0,0 +1,220 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Description:
+Inspired by https://github.com/pytorch/vision/pull/46
```

Comment:
can you give the reference to actual dataset or code in a github repo instead of a pull request link.

Reply:
Update the reference site 
https://github.com/apache/incubator-mxnet/pull/13966/commits/28740d2a1661d17e4f1b89c3c2a5052cdad97fd0